### PR TITLE
Feaure/pyro install support

### PR DESCRIPTION
**Pull request by KdVos**

### CODE REVIEW

File: setup.bash [hunk 1]
```diff
@@ -16,13 +16,16 @@ then
...
             echo "[emc-env] Detected ubuntu 20.04, using ROS Noetic"
             ;;
+       "18.04")
+            EMC_ROS_DISTRO=melodic
+            echo "[emc-env] Detected ubuntu 18.04, using ROS Melodic"
+            ;;
         *)
-            echo "[emc-env] Ubuntu $DISTRIB_RELEASE is unsupported. Use either 20.04, 20.04 or 20.04"
+            echo "[emc-env] Ubuntu $DISTRIB_RELEASE is unsupported. Use 20.04, 18.04"
             exit 1
```
Comment (KdVos) [inline setup.bash:29]: @MatthijsBurgh we need to have a look at this

File: setup/emc-update-impl.bash [hunk 2]
```diff
@@ -127,6 +127,7 @@ then
     _git_clone_or_update https://github.com/husarion/rosbot_description.git "$EMC_SYSTEM_DIR"/src/rosbot_description
 else
     # Robot specific packages
+    _git_clone_or_update https://github.com/tue-robotics/rosbot_bringup.git "$EMC_SYSTEM_DIR"/src/rosbot_bringup
     _git_clone_or_update https://github.com/tue-robotics/mrc_hero_bringup "$EMC_SYSTEM_DIR"/src/mrc_hero_bringup
```
Comment (MatthijsBurgh) [inline setup/emc-update-impl.bash:131]: Don't install both, when you only need one...

Comment (KdVos) [reply setup/emc-update-impl.bash:131]: I know, that's one of the (other) reasons it was still a draft. In order to select which to install we need to set the appropriate enviroment variables, which hasn't been done yet.

File: setup.bash [hunk 3]
```diff
@@ -16,13 +16,16 @@ then
     fi
 
     # Set ROS version
-    case $DISTRIB_RELEASE in
-        "20.04")
+    case $DISTRIB_RELEASE in "20.04")
```
Comment (KdVos) [inline setup.bash:19]:
```suggestion
    case $DISTRIB_RELEASE in 
    "20.04")
```

**Review by PetervDooren — DISMISSED** [review]

**Review by PetervDooren — APPROVED** [review]

**Pull request merged**